### DISCUSSION TITLE
Jellyfin crashes when trying to load one of the libraries on the App

### DISCUSSION THREAD

**Discussion by ZeekWorld** [jellyfin/jellyfin#8471] [General] [answered]
I am running Jellyfin on Docker, configured three libraries, two of them, Movies and TV Shows no problems, but when I try to open the third folder of also a movie library Jellyfin crashes every time. Tried to delete the library, re added it without luck. Restarted docker, etc.
This is happening on more than one Google TV media player so I would think is something to do with the server or the library. Loading the same library on my computer via browser works just fine.
I am not experts on reading the crash report. Crash logs attached.
Can one of you please take a look and let me know if there is an easy fix.
[Log.txt](https://github.com/jellyfin/jellyfin/files/9668415/Log.txt)

Thank you in advance.
Zeek

**Comment by nielsvanvelzen** [comment, marked as answer]
This crash is caused by a change in 10.8.5 that requires updates for our (Android-)apps. Until these updates are ready (of which we have no ETA) you can downgrade to 10.8.4.

**Comment by ZeekWorld** [comment]
I will downgrade and give it a try, thanks for the quick response!

**Reply by ZeekWorld** [reply]
Downgrading to 10.8.4 fixed the issue.
